packages/platform/atoms/booker/BookerPlatformWrapper.tsx

```diff
@@ -88,6 +92,14 @@ export const BookerPlatformWrapper = (props: BookerPlatformWrapperAtomProps) =>
       setOrg(null);
       setSelectedMonth(null);
       setSelectedDuration(null);
+      if (props.rescheduleUid) {
+        // clean booking data from cache
+        queryClient.removeQueries({
+          queryKey: [BOOKING_RESCHEDULE_KEY, props.rescheduleUid],
```

Comment:
Question: how do we know the `queryKey` ? Should it by any chance reflect `const key` in [useInitialFormValues](https://github.com/calcom/cal.com/pull/14945/files#diff-3c09f208072b8ff9454fb42d439f1d9b8c9d90288a9abe1ca9f5a933d7ad787eR143) ?

Reply:
this is not directly related to the useInitialFormValues, this is to clean the data from react-query so that the booking data is fresh when the component is remounted